CMakeLists.txt

```diff
@@ -28,6 +28,17 @@ if ( ${CMAKE_BINARY_DIR} STREQUAL ${CMAKE_CURRENT_SOURCE_DIR})
   message(FATAL "In-source build is not allowed")
 endif ()
 
+project ("rocm-validation-suite")
```

Comment:
This should be line 27  of the file.
This make cmake give project umbrella to the variables / execution etc.
Hence this should be second line after the version check.

Reply:
Done